e2e_example/lib/transformer.dart

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-import 'package:build/build.dart';
-
-import 'copy_builder.dart';
-
-class CopyTransformer extends BuilderTransformer {
-  @override
-  final List<Builder> builders = [new CopyBuilder()];
-
-  CopyTransformer.asPlugin(_);
```

Comment:
do we want to keep an example like this one to illustrate how to plug a build transformer as a regular pub-transformer (which requires you to have a `asPlugin` constructor that pub can reflectively use to allocate the transformer)?

It can be simpler with your change though, for example:
```dart
class CopyTransformer extends BuildTransformer {
  CopyTransformer.asPlugin(_) : super([new CopyBuilder()]);
}
```